### ISSUE TITLE
Request new layer analyzing the turning radius  and gradient of mapped rail line

### ISSUE THREAD

**Issue by c933103** [OpenRailwayMap/OpenRailwayMap#694]
Given that all rail lines mapped in OpenStreetMap already have their geometry mapped, is it possible to automatically analyze and show the turning radius of each curve of all mapped rail line?
And then is it possible to use elevation data to figure out track gradient automatically?

**Comment by bigbug21** [comment]
Good idea! Such analysis, even if it is not very precise, would be a great help to seek for potentials in current infrastructure, for example higher speeds in curves.
